src/shim_ve2/xdna_hwctx.cpp

```diff
@@ -110,9 +121,7 @@ xdna_hwctx(const device_xdna& dev, const xrt::xclbin& xclbin, const xrt::hw_cont
   m_info = get_partition_info_hw(&m_device, xclbin.get_uuid(), arg.handle);
   set_doorbell(arg.umq_doorbell);
 
-  auto data = m_device.get_axlf_section(AIE_METADATA, xclbin.get_uuid());
-  if (data.first && data.second)
-    m_aie_array = std::make_shared<xdna_aie_array>(&m_device, this);
+  m_aie_array = std::make_shared<xdna_aie_array>(&m_device, this);
```

Comment:
dont we need to check for AIE_TRACE_METADATA before calling aie_array constructor?